package.json

```diff
@@ -37,7 +37,7 @@
     "css-loader": "^0.28.7",
     "enzyme": "^3.1.0",
     "enzyme-adapter-react-16": "^1.0.1",
-    "eslint": "^4.8.0",
+    "eslint": "^4.12.1",
```

Comment:
@mdiwale 
Where are these changes coming from?  I would have expected changes to _package.json_ or _yarn.lock_ in this PR.

Reply:
@thescientist13 After I cloned the project and opened it in WebStorm, I got a notification to upgrade yarn. Hence I ran the command ```yarn upgrade``` which probably led these changes. Otherwise, nothing else I changed.